### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by blag**
I wasn't sure what details to put in here, but somebody else might so I left in some comments (auto-generated by GitHub), just in case they inspire somebody else to fill this out a bit more.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+-->
```
Comment (jedcunningham) [inline SECURITY.md:15]:
```suggestion
# Security Policy

## Supported Versions

Details on which versions are still in maintenance can be found in the [AC Support Policy](https://docs.astronomer.io/software/ac-support-policy#astronomer-certified-lifecycle-schedule) documentation.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+Details on which versions are still in maintenance can be found in the [AC Support Policy](https://docs.astronomer.io/software/ac-support-policy#astronomer-certified-lifecycle-schedule) documentation.
+
+## Reporting a Vulnerability
+
+We understand that community members may identify potential security vulnerabilities.
+Identified vulnerabilities should be reported to security@astronomer.io.
```
Comment (jedcunningham) [inline SECURITY.md:10]: It might be worth making this more prominent?

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
@uchi-mata, can you take a look at this when you get a chance? Thanks.

**Review by uchi-mata — APPROVED** [review]
Looks good to me, thanks for adding this!

**Pull request merged**